internal/v2/s2av2_test.go

```diff
@@ -318,7 +318,18 @@ func (x s2ATestStream) Send(m *s2av2pb.SessionReq) error {
 }
 
 func (x s2ATestStream) Recv() (*s2av2pb.SessionResp, error) {
-	return nil, nil
+	return &s2av2pb.SessionResp{
```

Comment:
curious, why is this change needed? were the tests having issue? thx

Reply:
If you follow s2aTLSClientConfigFactory.Build() --> v2.NewClientTLSConfig() --> tlsconfigstore.GetTLSConfigurationForClient, it'll panic on nil ptr dereferences:
  * s2a-go/internal/v2/tlsconfigstore/tlsconfigstore.go:53
  * s2a-go/internal/v2/tlsconfigstore/tlsconfigstore.go:82

Reply:
SG